roles/build-release/tasks/build.yaml

```diff
@@ -4,18 +4,27 @@
     _deps_file: "ansible-{{ antsibull_ansible_version }}.deps"
     _release_archive: "{{ antsibull_sdist_dir }}/ansible-{{ antsibull_ansible_version }}.tar.gz"
 
+# Documentation for the following commands:
+# https://github.com/ansible-community/antsibull/blob/main/docs/build-ansible.rst
+
 - name: Update version ranges in the build file for alpha and beta releases
   command: >-
     poetry run coverage run -p --source antsibull -m antsibull.cli.antsibull_build new-ansible {{ antsibull_ansible_version }}
       --data-dir {{ antsibull_data_dir }}
   when: antsibull_ansible_version is regex("^\d+.\d+.\d+(a\d+|b1)$")
 
+- name: Set up feature freeze for b2 through rc1
+  set_fact:
+    _feature_freeze: "--feature-frozen"
+  when: antsibull_ansible_version is regex("^\d+.\d+.\d+(b2|b3|rc1)$")
```

Comment:
This is incorrect since we'd want the first release (i.e, after release candidate) to be frozen as well... I suppose we could match against ``\d+\.0\.0`` too ?